charts/ndm/values.yaml

```diff
@@ -0,0 +1,98 @@
+# Default values for ndm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+release:
+  version: "0.8.0"
+ndm:
+  componentName: ndm
+  enabled: true
+  image:
+    # Make sure that registry name end with a '/'.
+    # For example : quay.io/ is a correct value here and quay.io is incorrect
+    registry:
```

Comment:
should the value be `""` or can it be left empty?

Reply:
Both are fine but putting `""` gives more idea re the placeholder. Will fix it. Thanks.